Macbook-setup.md

```diff
@@ -0,0 +1,163 @@
+# Mac - new system setup instructions
+
+These instructions describe software installations for MacOS as of 6/24/2021 and are based on an Intel-based Mac running Big Sur (macOS v11.4). Any problems with this process may result from a different architecture, OS version, or newer versions of the software being installed.
+
+_Note: MacOS uses Z shell (zsh) as the default shell. If you prefer bash, you can make that the default shell under System Preferences > Users and Groups (unlock the lock icon to make changes) > right click your user, select 'Advanced Options' and change 'Login shell' to /bin/bash. Then use ~/.bashrc instead of ~/.zshrc throughout this setup._
+
+## Download and install the following software:
+ - [Slack](https://slack.com/downloads/mac)
+ - [Zoom](https://zoom.us/download)
+ - [OneDrive](https://www.microsoft.com/en-us/microsoft-365/onedrive/download)*
+ - [Chrome](https://www.google.com/chrome/)*
+ - [LastPass for Chrome](https://lastpass.com/misc_download2.php)
```

Comment:
Not a blocker, but perhaps:

 `- Browser, such as [Chrome](https://www.google.com/chrome/)
 `
 Doesn't need to be Chrome specifically ;)

Reply:
Updated.  Also did the same for text editor (doesn't need to be VSCode specifically)